doc/containers.rst

```diff
@@ -0,0 +1,46 @@
+.. _containers:
+
+Containers
+==========
+
+We provide a pyPESTO docker image in OCI format through `docker.io/stephanmg/pypesto`.
+
+Develop versions are pushed as tag ``latest``.
+
+Docker image can also be build manually by the users using the Dockerfile: https://github.com/ICB-DCM/pyPESTO/tree/main/docker
+
+If you manually build your container and need to convert to Apptainer, first export your docker image to `.tar` by:
+`docker save -o pypesto.tar pypesto:latest` and pull the image before from docker.io by `docker pull docker.io/stephanmg/pypesto:latest`.
```

Comment:
Why should one go through docker if one wants to use apptainer? `apptainer pull docker://stephanmg/pypesto:latest`? 


"first export ... pull before..." is somewhat unclear.

Reply:
Fixed.